pandas/tests/io/json/test_normalize.py

```diff
@@ -85,7 +85,52 @@ def missing_metadata():
     ]
 
 
-class TestJSONNormalize:
+@pytest.fixture
+def deep_nested_inconsistantly_formatted():
```

Comment:
Can you simplify this? I don't think `keyA` and `keyB` are ever hit in tests so would be preferable to remove them

Reply:
Removed